lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -222,12 +222,20 @@ def self.ruby_method_runnable?(aem)
     end
 
     def self.setup_drb_for_ruby_method
+      require 'drb/timeridconv'
+      DRb.install_id_conv DRb::TimerIdConv.new drb_cache_timeout
       drb_front  = MiqAeMethodService::MiqAeServiceFront.new
       drb        = DRb.start_service("druby://127.0.0.1:0", drb_front)
     end
 
+    def self.drb_cache_timeout
+      3600 # 1 hour
```

Comment:
@mkanoor Can you just make this one change?  Then I think this is good to merge.  You can probably make this a constant, but I assume you are keeping as a method for testing purposes?